etl/steps/data/garden/faostat/2022-05-17/faostat_food_explorer.py

```diff
@@ -0,0 +1,483 @@
+    "Meat, horse",
+    "Meat, lamb and mutton",
+    "Meat, mule",
+    "Meat, pig",
+    "Meat, poultry",
+    "Meat, rabbit",
+    "Meat, sheep and goat",
+    "Meat, turkey",
+    "Melon",
+    "Melonseed",
+    "Milk",
+    "Millet",
+    "Mixed grains",
+    "Molasses",
+    "Mushrooms",
+    "Mustard seed",
+    "Nuts",
+    "Oats",
+    "Offals",
+    "Offals, buffaloes",
+    "Offals, camels",
+    "Offals, cattle",
+    "Offals, goats",
+    "Offals, horses",
+    "Offals, pigs",
+    "Offals, sheep",
+    "Oilcrops",
+    "Oilcrops, Cake Equivalent",
+    "Oilcrops, Oil Equivalent",
+    "Okra",
+    "Olive oil",
+    "Olives",
+    "Onions",
+    "Oranges",
+    "Palm fruit oil",
+    "Palm kernel oil",
+    "Palm kernels",
+    "Palm oil",
+    "Papayas",
+    "Peaches and nectarines",
+    "Pears",
+    "Peas, dry",
+    "Peas, green",
+    "Pepper",
+    "Pigeon peas",
+    "Pineapples",
+    "Pistachios",
+    "Plantains",
+    "Plums",
+    "Poppy seeds",
+    "Pork",
+    "Potatoes",
+    "Pulses",
+    "Quinoa",
+    "Rapeseed",
+    "Rapeseed oil",
+    "Raspberries",
+    "Rice",
+    "Roots and tubers",
+    "Rye",
+    "Safflower oil",
+    "Safflower seed",
+    "Seed cotton",
+    "Sesame oil",
+    "Sesame seed",
+    "Silk",
+    "Skins, goat",
+    "Skins, sheep",
+    "Sorghum",
+    "Soybean oil",
+    "Soybeans",
+    "Spinach",
+    "Strawberries",
+    "String beans",
+    "Sugar (raw)",
+    "Sugar beet",
+    "Sugar cane",
+    "Sugar crops",
+    "Sunflower oil",
+    "Sunflower seed",
+    "Sweet potatoes",
+    "Tangerines",
+    "Tea",
+    "Tobacco",
+    "Tomatoes",
+    "Total",
+    "Treenuts",
+    "Vegetables",
+    "Walnuts",
+    "Watermelons",
+    "Wheat",
+    "Whey",
+    "Wine",
+    "Wool",
+    "Yams",
+]
+# OWID item name, element name, and unit name for population (as given in faostat_qcl and faostat_fbsc datasets).
+FAO_POPULATION_ITEM_NAME = "Population"
+FAO_POPULATION_ELEMENT_NAME = "Total Population - Both sexes"
+FAO_POPULATION_UNIT = "1000 persons"
+
+
+def combine_qcl_and_fbsc(
+    qcl_table: catalog.Table, fbsc_table: catalog.Table
+) -> pd.DataFrame:
+    """Combine garden `faostat_qcl` and `faostat_fbsc` datasets.
+
+    Parameters
+    ----------
+    qcl_table : catalog.Table
+        Main table (in long format) of the `faostat_qcl` dataset.
+    fbsc_table : catalog.Table
+        Main table (in long format) of the `faostat_fbsc` dataset.
+
+    Returns
+    -------
+    combined : pd.DataFrame
+        Combined data (as a dataframe, not a table).
+
+    """
+
+    columns = [
+        "country",
+        "year",
+        "item_code",
+        "element_code",
+        "item",
+        "element",
+        "unit",
+        "unit_short_name",
+        "value",
+        "population_with_data",
+    ]
+    qcl = pd.DataFrame(qcl_table).reset_index()[columns]
+    qcl["value"] = qcl["value"].astype(float)
+    qcl["element"] = [element for element in qcl["element"]]
```

Comment:
Note - even though `qcl["element"].astype(str)` is slow, `qcl["element"].astype("string")` seems to be even faster than list comprehension (there might be more problems with "string" type down the road though)